readme.md

```diff
@@ -47,37 +61,217 @@ docker run \
 | --------- | ------------- | --------------------------------------------------------------------------------------------------------- |
 | NODE_ENV  |               | node environment setup                                                                                    |
-| LOG_LEVEL | info          | [pino log levels](https://github.com/pinojs/pino/blob/master/docs/api.md#loggerlevel-string-gettersetter) |
+| LOG_LEVEL | `info`        | [pino log levels](https://github.com/pinojs/pino/blob/master/docs/api.md#loggerlevel-string-gettersetter) |
 | HTTP_HOST | `0.0.0.0`     |                                                                                                           |
 | HTTP_PORT | `3000`        |                                                                                                           |
 
+### Routes
+
+| Method | Path                       | Description                                                                              |
+| ------ | -------------------------- | ---------------------------------------------------------------------------------------- |
+| `POST` | `/v1/decoder`              | General decoder route. Requires to specify [protocol](#protocols) in the requests body.  |
+| `POST` | `/v1/decoder/analog`       | Decoder for the `analog` protocol based devices.                                         |
+| `POST` | `/v1/decoder/mtx`          | Decoder for the `mtx` protocol based devices.                                            |
+| `POST` | `/v1/decoder/mtxLora`      | Decoder for the `mtxLora` protocol based devices.                                        |
+| `POST` | `/v1/decoder/obisObserver` | Decoder for the `obisObserver` protocol based devices.                                   |
+| `POST` | `/v1/encoder`              | General encoder route. Requires to specify [protocol](#protocols)  in the requests body. |
+| `POST` | `/v1/encoder/analog`       | Encoder for the `analog` protocol based devices.                                         |
+| `POST` | `/v1/encoder/mtx`          | Encoder for the `mtx` protocol based devices.                                            |
+| `POST` | `/v1/encoder/mtxLora`      | Encoder for the `mtxLora` protocol based devices.                                        |
+| `POST` | `/v1/encoder/obisObserver` | Encoder for the `obisObserver` protocol based devices.                                   |
+
+
+### POST request parameters
+
+#### Framing format
+
+| Name   | Value | Description         |
+| ------ | ----- | ------------------- |
+| `NONE` | `0`   | no framing          |
+| `HDLC` | `1`   | `HDLC` frame format |
+
+Default value: `0`
+Example: `framingFormat: 1`
+
+#### Bytes conversion format
+
+| Name     | Value | Description                   |
+| -------- | ----- | ----------------------------- |
+| `HEX`    | `1`   | data treats as hex string     |
+| `BASE64` | `2`   | data threats as base64 string |
+
+Default value: `1`
```

Comment:
```suggestion
Default value: `1`<br>
```
in case you need 2 lines here

and maybe use dots at the end

Reply:
fixed